src/components/card/CategoryCard.css

```diff
@@ -0,0 +1,43 @@
+.category-card-container{
+    width:20rem;
+    height:30rem;
+    position:relative;
+}
+.category-card-img{
+    display:block;
+    width:100%;
+    height:75%;
+    object-fit:cover; 
+}
+.text-overlay-container{
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
```

Comment:
If in case the need arises to provide a top:0 attribute too then instead of using all the 4 properties you can use a single property like `inset:0`.

Reply:
ok will use it